oracledb-ballerina/tests/00_init_container.bal

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
check license year in all files.

Reply:
Fixed with ae9c94747102126fe586ad09fa1f73d0652e8fcd